src/components/MapGuide/index.tsx

```diff
@@ -1,101 +1,74 @@
-import { useEffect, useRef, useState } from 'react';
 import Button from '@mui/material/Button';
-import DialogContent from '@mui/material/DialogContent';
-import DialogContentText from '@mui/material/DialogContentText';
-import Paper, { PaperProps } from '@mui/material/Paper';
-import Backdrop, { BackdropProps } from '@mui/material/Backdrop';
-import Draggable from 'react-draggable';
-import CloseIcon from '@mui/icons-material/Close';
+import { styled } from '@mui/material/styles';
+import Tooltip, { TooltipProps, tooltipClasses } from '@mui/material/Tooltip';
 
 import {
-    StyledMapGuideDialog,
     StyledMapGuideButton,
-    StyledDialogTitle,
-    CloseButton,
+    StyledTooltipTitle,
+    StyledTooltipContainer,
 } from './styled';
 
-const PaperComponent = (props: PaperProps) => {
+import React from 'react';
+
+const StyledMapGuideTooltip = styled(
+    ({ className, ...props }: TooltipProps) => (
+        <Tooltip {...props} classes={{ popper: className }} />
+    ),
+)(({ theme }) => ({
+    [`& .${tooltipClasses.tooltip}`]: {
+        backgroundColor: '#0094e2',
```

Comment:
You should use `theme` here to set a color. This way whenever a new theme colors are needed it can be changed only in `src/theme/theme.ts` rather than in individual files